lib/rucio/common/constants.py

```diff
@@ -71,16 +72,6 @@ class SuspiciousAvailability(enum.Enum):
     LAST_COPY = 2
 
 
-class ReplicaState(enum.Enum):
```

Comment:
Is this just not used anywhere?

Reply:
It's used a lot of places, just moved to `rucio.common.types.enums` (and imported from there into `constants.py` so I don't have to change imports all over the codebase).